linux/debian/rules

```diff
@@ -20,9 +20,9 @@ export LC_ALL=C.UTF-8
 override_dh_auto_configure:
 	# keymankeyboardprocessor
 	cd core && \
-		./build.sh configure -- --wrap-mode=nodownload --prefix=/usr --sysconfdir=/etc \
+		./build.sh configure --configure="--wrap-mode=nodownload --prefix=/usr --sysconfdir=/etc \
 			--localstatedir=/var --libdir=lib/$(DEB_TARGET_MULTIARCH) \
-			--libexecdir=lib/$(DEB_TARGET_MULTIARCH)
+			--libexecdir=lib/$(DEB_TARGET_MULTIARCH)"
```

Comment:
I can't find why this was made… the `--` form seems cleaner, but there must be some reason…?

Reply:
Back to using `--` form again now.